npm i -D  tailwindcss-vite-plugin gives ERESOLVE unable to resolve dependency tree

Hi, I'm trying to install this plugin but npm complains about the following:

   ~/r/f/vite-01  on   main !1  npm i -D tailwindcss-vite-plugin             ✔  at 15:09:41  
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: vite-01@0.0.0
npm ERR! Found: vite@4.1.4
npm ERR! node_modules/vite
npm ERR!   dev vite@"^4.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^3.1.7" from tailwindcss-vite-plugin@0.0.3
npm ERR! node_modules/tailwindcss-vite-plugin
npm ERR!   dev tailwindcss-vite-plugin@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/xyz/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/xyz/.npm/_logs/2023-03-14T14_10_08_966Z-debug-0.log

    ~/r/f/vite-01  on   main !1      

I should not need to use --force, or --legacy-peer-deps, correct? Should this be fixed upstream?

Thanks!

--- a/examples/next.js/package.json
+++ b/examples/next.js/package.json
@@ -7,8 +7,8 @@
   },
   "dependencies": {
     "next": "latest",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18",
+    "react-dom": "^18"
   },
   "devDependencies": {
     "@types/node": "17.0.4",

--- a/examples/vite-react/package.json
+++ b/examples/vite-react/package.json
@@ -16,7 +16,7 @@
     "@types/react-dom": "^18.0.6",
     "@vitejs/plugin-react": "^2.1.0",
     "typescript": "^4.6.4",
-    "vite": "^3.1.0",
+    "vite": "^4",
     "tailwindcss": "^3.1.8",
     "autoprefixer": "^10.4.12",
     "postcss": "^8.4.17",

--- a/packages/utils/package.json
+++ b/packages/utils/package.json
@@ -1,6 +1,6 @@
 {
   "name": "tailwindcss-webpack-plugin-utils",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "common utils for tailwindcss-webpack-plugin and tailwindcss-vite-plugin",
   "main": "./dist/index.js",
   "module": "./dist/index.mjs",

--- a/packages/vite-plugin/package.json
+++ b/packages/vite-plugin/package.json
@@ -1,5 +1,5 @@
 {
-  "version": "0.0.3",
+  "version": "0.0.5",
   "name": "tailwindcss-vite-plugin",
   "license": "MIT",
   "description": "Design in Devtools for Tailwind CSS in Vite",
@@ -39,7 +39,7 @@
     "dev": "pnpm run build --watch"
   },
   "peerDependencies": {
-    "vite": "^3.1.7"
+    "vite": "^3 || ^4"
   },
   "dependencies": {
     "tailwindcss": "^3.1.8",
@@ -49,6 +49,6 @@
   "devDependencies": {
     "@types/tailwindcss": "^3.1.0",
     "tsup": "^6.2.3",
-    "vite": "^3.1.7"
+    "vite": "^4"
   }
 }

